### DISCUSSION TITLE
Add a satellite and streets Base Map

### DISCUSSION THREAD

**Discussion by indranildeveloper** [keplergl/kepler.gl#2730] [Ideas]
Currently, Kepler GL does not have a satellite and streets base map. If possible I would like to add these two base maps:

1. Satellite Map: https://api.maptiler.com/maps/satellite/style.json?key=<API_KEY>
2. Street Map: https://api.maptiler.com/maps/streets-v2/style.json?key=<API_KEY>

@igorDykhta @heshan0131 @ibgreen  Is that something we want?

**Comment by igorDykhta** [comment]
@indranildeveloper We will add back Mapbox support in the next release with dynamic map library switching.

**Reply by indranildeveloper** [reply]
Ok, Awesome 💯
